doc/nrf/ug_bt_mesh_concepts.rst

```diff
@@ -0,0 +1,318 @@
+.. _mesh_concepts:
+
+Bluetooth Mesh concepts
+#######################
+
+Bluetooth Mesh is a profile specification developed and published by the Bluetooth Special Interest Group (SIG).
+This document explains the basic concepts of the Bluetooth Mesh and gives an overview of the operation and capabilities of the profile, as well as the life cycle of a mesh device.
+For more information about Nordic Semiconductor's implementation of the Bluetooth Mesh, see :ref:`Bluetooth Mesh architecture documentation <mesh_architecture>`.
+
+The Bluetooth Mesh is based on the |BLE| part of the Bluetooth 4.0 Specification, and shares the lowest layers with this protocol.
+On-air, the Bluetooth Mesh physical representation is compatible with existing |BLE| devices, as mesh messages are contained inside the payload of |BLE| *advertisement* packets.
+However, Bluetooth Mesh specifies a completely new host layer, and although some concepts are shared, Bluetooth Mesh is incompatible with the |BLE| host layer.
+
+.. figure:: /images/bt_mesh_and_ble.svg
+   :alt: Relationship between Bluetooth Mesh and |BLE| specifications
+
+   Relationship between Bluetooth Mesh and |BLE| specifications
+
+Read more about basic Bluetooth Mesh concepts in the following sections:
+
+* :ref:`Application areas <mesh_concepts_app_areas>`
+* :ref:`Network topology and relaying <mesh_concepts_network_topo>`, including information about :ref:`transport <mesh_concepts_network_topo_transport>`, :ref:`relays <mesh_concepts_network_topo_relays>`, :ref:`power consumption <mesh_concepts_network_topo_power>`, and :ref:`GATT proxy <mesh_concepts_network_topo_gatt>`.
```

Comment:
Perhaps the subsubsections should be a list too?

Reply:
I would use here:
```
.. contents::
    :local:
    :depth: 2
```
This will cover both section and subsection names.